index.js

```diff
@@ -11,6 +11,7 @@ var homeOrTmp = require('home-or-tmp');
 
 var DEFAULT_IGNORE = [
 	'node_modules/**',
+	'**/node_modules/**',
```

Comment:
Trailing `/**` matches all directories and subdirectories. `/**/*` matches all files in all directories and subdirectories. 

In this case, they're the same. The prior is simpler. The former would be to match a certain file type: 
`/**/*.js`